flink-tests/src/test/java/org/apache/flink/test/streaming/runtime/BigUserProgramJobSubmitITCase.java

```diff
@@ -43,40 +40,27 @@
 /**
  * Integration test that verifies that a user program with a big(ger) payload is successfully
  * submitted and run.
+ *
+ * <p>This test fails for non-flip6 cluster due to job submission payload being too large, see [FLINK-7285].
  */
-@Ignore("Fails on job submission payload being too large - [FLINK-7285]")
+@Category(Flip6.class)
```

Comment:
this test actually succeeds with Flip6 so i figured we may just as well enable it.

Reply:
Ah, and you're marking it as `Flip6` because it doesn't succeed with the legacy components?